src/ui/views/Approval/components/TxComponents/GasSelecter.tsx

```diff
@@ -142,41 +142,21 @@ const GasSelector = ({
       }
     }
   };
-
-  useEffect(() => {
-    formValidator();
-  }, [customGas, afterGasLimit, selectedGas, gasList]);
-
   const loadGasMarket = async () => {
     const list = await wallet.openapi.gasMarket(
       chain.serverId,
       customGas && customGas > 0 ? Number(customGas) * 1e9 : undefined
     );
-    setGasList(
+    await setGasList(
```

Comment:
state hook is asynchronous but not return a Promise so this await is useless right?

Reply:
done